react/components/SelectionListItem.js

```diff
@@ -19,7 +19,7 @@ const SelectionListItem = ({ option, onItemClick, selected }) => {
   return (
     <button
       className={classNames(
-        handles.orderByOptionItem,
+        applyModifiers(handles.orderByOptionItem, selected && 'selected'),
```

Comment:
While this does work and is a very clever way of using `applyModifiers`, it wasn't created to be used without passing a valid string or string[] as the second argument:

![image](https://user-images.githubusercontent.com/8443580/87308962-5a0aac00-c4f2-11ea-9243-365286f7a34d.png)

I think it would be better to check `selected` previously so you only use `applyModifiers` if its value is `true`!

Reply:
@iaronaraujo take a look if it is better now

Reply:
It is! Thank you!